src/workspacer.Bar/Widgets/TitleWidget.cs

```diff
@@ -9,6 +9,8 @@ namespace workspacer.Bar.Widgets
     public class TitleWidget : BarWidgetBase
     {
         public Color MonitorHasFocusColor { get; set; } = Color.Yellow;
+        public bool IsShortTitle { get; set; } = false;
+        public string NoWindowMessage { get; set; } = "No Window";
```

Comment:
This text used to be "No window**s**" (plural).

Reply:
fixed in last commit, hadn't noticed